.github/workflows/validations.yaml

```diff
@@ -27,3 +27,15 @@ jobs:
 
       - name: Run all validations
         run: make pr-validations
+
+  WindowsValidations:
+    name: "Windows units"
+    runs-on: windows-2022
+    steps:
+      - uses: actions/checkout@v3
+
```

Comment:
Note that this doesn't use the caching that we'd get from using our own `actions/bootstrap` here, but that has some entries that require `shell: bash` which presumably won't work on Windows, and I don't know why it needs those yet.